addons/dialog/index.jsx

```diff
@@ -10,11 +10,23 @@ module.exports = React.createClass({
     }
   },
 
+  getTitle() {
+    let { title } = this.props
+    if (this.props.title) {
+      return (
+        <h3 className="col-dialog-title">
+          { this.props.title }
+        </h3>
```

Comment:
What if the `h3` were customizable here with like a `headingLevel` or something?